data/coalition/coalition missions.txt

```diff
@@ -3171,3 +3171,539 @@ mission "Heliarchs Buy Jump Drive Mission"
+	on offer
+		conversation
+			`Checking your computer feed for the latest galactic news and trade prices, you get a reminder from your computer.`
+			`	You read it, and recall that you had helped an author named Anna Fangvin get her book, 'Beyond the Hyperdrive,' published about half a year ago in Coalition space. You should return to <destination> and check on her.`
+				accept
+	on complete
+		payment 50000
+		"salary: Beyond the Hyperdrive" = 100
+		conversation "anna invites you to book signing"
+
+mission "Coalition: Struggling Author 5B"
+	landing
+	source
+		near "Quaru" 0 100
+	destination "Inmost Blue"
+	to offer
+		has "event: beyond the hyperdrive is published"
+		not "Coalition: Struggling Author 5A: offered"
+	on offer
+		conversation
+			`As you land on the spaceport, you check your messages. You see a message from Anna Fangvin, the author who you helped about half a year ago. In the message, she explains that her novel, 'Beyond the Hyperdrive,' has been approved. You should return to <destination> and meet her.`
+				accept
+	on complete
+		payment 50000
+		"salary: Beyond the Hyperdrive" = 100
+		conversation "anna invites you to book signing"
+
+
+conversation "anna invites you to book signing"
+	action
+		log "People" "Anna Fangvin" `Anna is a science fiction writer. She was a struggling author in human space, but found success upon publishing her novel 'Beyond the Hyperdrive' in Coalition space. Her book is a bestseller in Kimek space.`
+		log "Helped somebody named Anna Fangvin get her book published in Coalition space. Apparently, it's a bestseller now. The book was 'Beyond the Hyperdrive', something about humans stealing jump drives from the Quarg. Also found out she was ambidextrous."
+		event "beyond the hyperdrive payment" 40
+		set "beyond the hyperdrive news can show now"
+	`The moment you land, Anna is all too happy to see you. She grabs your shoulders with both arms and in an excited tone explains that her book has been approved for publication.`
+	`	"They're publishing 'Beyond the Hyperdrive' today, and there's a full release event where I'll be signing copies! Thanks for everything you've done, <first>!"`
+	choice
+		`	"It got approved? That's awesome!"`
+	`	"I know, right? So do you want to join me during the book signing?"`
+	choice
+		`	"I've been with you through the whole process. Count me in!"`
+	`	"Great! Let's walk to the bookstore and get ready! The staff have already prepared seats for both of us."`
+	`	You and Anna walk into the massive bookstore at the spaceport. There are already several Heliarch present, their Enforcer Riot Staffs in place for crowd control.`
+	`	A Kimek, dressed in what looks like the bookstore's uniform, takes out the Kimek equivalent of a loudspeaker and begins announcing the release of 'Beyond the Hyperdrive'. Within minutes, the bookstore has become crowded, with a long queue forming between the cashiers and your table, as Anna signs every copy brought to her by Kimek.`
```

Comment:
'Kimek equivalent of a loudspeaker'? Why not just a loudspeaker? What is it about this object that makes it different from a human loudspeaker? Not to sound harsh but '[alien] equivalent of [normal human object]' is kind of a lazy way of describing something.